server/api.js

```diff
@@ -30,9 +31,118 @@ router.get("/energisers", (req, res) => {
     }).catch((err) => {
         console.log(err);
         res.status(500).send(err);
+=======
```

Comment:
And here

Reply:
I'm not sure how did it get there 😄 
Shall I remove these ==== signs in VS code and commit again?